spec/datetimeformat.html

```diff
@@ -186,6 +186,19 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
             1. Let _rangePatterns_ be _bestFormat_.[[rangePatterns]].
         1. Set _dateTimeFormat_.[[Pattern]] to _pattern_.
         1. Set _dateTimeFormat_.[[RangePatterns]] to _rangePatterns_.
+        1. Let _nf1Options_ be OrdinaryObjectCreate(*null*).
+        1. Perform ! CreateDataPropertyOrThrow(_nf1Options_, *"useGrouping"*, *false*).
+        1. Set _dateTimeFormat_.[[NumericNumberFormat]] to ? Construct(%NumberFormat%, &laquo; _dateTimeFormat_.[[Locale]], _nf1Options_ &raquo;).
```

Comment:
If these uses of Construct could actually throw, then the PR would be normative. But I don't see how they can.
```suggestion
        1. Set _dateTimeFormat_.[[NumericNumberFormat]] to ! Construct(%NumberFormat%, &laquo; _dateTimeFormat_.[[Locale]], _nf1Options_ &raquo;).
```